lib/remote_system.rb

```diff
@@ -140,4 +140,8 @@ def read_file(file)
       raise
     end
   end
+
+  def inject_file(local_file, remote_path)
+    system("scp #{local_file} root@#{host}:#{remote_path}")
```

Comment:
This method is just a stub for the sake of trying the prototype. You are right, that it needs to be implemented properly.